src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.cs

```diff
@@ -0,0 +1,151 @@
+#if MAUI
+        public static readonly BindableProperty ElementProperty =
+            BindableProperty.Create(nameof(Element), typeof(TextFormElement), typeof(TextFormElementView), null, propertyChanged: (s, oldValue, newValue) => ((TextFormElementView)s).OnElementPropertyChanged(oldValue as TextFormElement, newValue as TextFormElement));
+#else
+        public static readonly DependencyProperty ElementProperty =
+            DependencyProperty.Register(nameof(Element), typeof(TextFormElement), typeof(TextFormElementView), new PropertyMetadata(null, (s,e) => ((TextFormElementView)s).OnElementPropertyChanged(e.OldValue as TextFormElement, e.NewValue as TextFormElement)));
+#endif
+
+        private void OnElementPropertyChanged(TextFormElement? oldValue, TextFormElement? newValue)
+        {
+            if (oldValue is INotifyPropertyChanged inpcOld)
+            {
+                _elementPropertyChangedListener?.Detach();
+                _elementPropertyChangedListener = null;
+            }
+            if (newValue is INotifyPropertyChanged inpcNew)
+            {
+                _elementPropertyChangedListener = new WeakEventListener<TextFormElementView, INotifyPropertyChanged, object?, PropertyChangedEventArgs>(this, inpcNew)
+                {
+                    OnEventAction = static (instance, source, eventArgs) => instance.Element_PropertyChanged(source, eventArgs),
+                    OnDetachAction = static (instance, source, weakEventListener) => source.PropertyChanged -= weakEventListener.OnEvent,
+                };
+                inpcNew.PropertyChanged += _elementPropertyChangedListener.OnEvent;
+            }
+            UpdateText();
+            UpdateVisibility();
+        }
+
+        private void UpdateVisibility()
+        {
+            if (_readonlyLabel != null)
+            {
+#if MAUI
+                _readonlyLabel.IsVisible = Element?.IsVisible == true;
+#else
+                _readonlyLabel.Visibility = Element?.IsVisible == true ? Visibility.Visible : Visibility.Collapsed;
+#endif
+            }
+        }
+
+        private void UpdateText()
+        {
+            if (_readonlyLabel is not null)
+            {
+                string text = "";
+                if (Element is not null)
+                {
+                    text = Element.Format == FormTextFormat.PlainText ? Element.Text : RemoveMarkdown(Element.Text);
+                }
+                _readonlyLabel.Text = text;
+            }
+        }
+
+        private static string RemoveMarkdown(string markdownText)
+        {
+            return Regex.Replace(markdownText, @"(\*\*|__)(.*?)\1", "$2")
+                        .Replace("*", "")
+                        .Replace("_", "")
+                        .Replace("`", "")
+                        .Replace("~", "")
+                        .Replace("<br>", "\n")
+                        .Replace("<br/>", "\n")
+                        .Replace("\n", Environment.NewLine);
+        }
```

Comment:
This is not great &mdash; you'd need a dozen carefully-ordered regexes to cover markdown syntax &mdash; but I understand that it's a placeholder for something better.

Reply:
Yes I'm aware - it'll be gone before we ship.